digit/server.go

```diff
@@ -36,4 +37,5 @@ func (s *Server) configure() {
 	//	s.Static("/", "./templates/open-digit/dist/")
 	s.LoadHTMLGlob("templates/open-digit/dist/*.html")
 	s.GET("/", s.Index)
+	s.POST("/plaid/exchangeToken", s.ExchangeToken)
```

Comment:
I'm setting up an endpoint for the exchange token here. It doesn't do anything yet, but just FYI for the future. Essentially once you get the access token, you can post it here with a JSON post body that we can agree upon later. I'm thinking something like

POST /plaid/exchangeToken
```json
{ "publicToken": "somepublictokenhere" }
```